packages/fast-usdc/src/exos/settler.js

```diff
@@ -94,13 +95,15 @@ export const prepareSettler = (
      *   remoteDenom: Denom;
      *   repayer: LiquidityPoolKit['repayer'];
      *   settlementAccount: HostInterface<OrchestrationAccount<{ chainId: 'agoric' }>>
-     *   intermediateRecipient: ChainAddress;
+     *   intermediateRecipient?: ChainAddress;
      * }} config
      */
     config => {
       log('config', config);
       return {
         ...config,
+        // make sure the state record has this property, perhaps with an undefined value
```

Comment:
consider using "the state **schema**" to suggest the implications

Reply:
The schema is below in `stateShape`, right? That's how I knew to write the code this way: it refused to store the state record without this property.